data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,41 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
+    "unmanagedWrappingKeys": {
```

Comment:
Bit nitpicky, but is `unmanaged` appropriate here? 
I think I know you mean not managed by Corda, but it may sound wrong.
Maybe we should just call it `wrappingKeys` to keep it simple?

I know naming is difficult ;)

Reply:
Possibly this should be called "wrappingKeys" or "externalWrappingKeys". TBD more widely.